pontoon/sync/formats/__init__.py

```diff
@@ -20,6 +20,7 @@
     '.ini': silme.parse_ini,
     '.inc': silme.parse_inc,
     '.l20n': l20n.parse,
+    '.ftl': ftl.parse,
```

Comment:
I guess it would be good idea to add a small comment about the future of support l20n format.